### ISSUE TITLE
Images need alt attributes

### ISSUE THREAD

**Issue by oliverjam** [fac21/studio-pets#22]
Images need alternative descriptions ("alt text") so that people who cannot see the image still get the same information communicated. For example a low-vision user using a screen reader to browse will just hear "/cat.jpg" if there's no alt. This is also what appears if the image fails to load for some reason.

If an image is entirely decorative (for example a fancy line to divide sections) then you can use an empty alt attribute to tell screen readers to skip the image (`alt=""`)
